css/properties.schema.json

```diff
@@ -26,6 +26,7 @@
         "integer",
         "length",
         "lpc",
+        "no",
```

Comment:
Oh, also: it seems that where the spec has "Animatable: no", we use "discrete". See for example:
https://developer.mozilla.org/en-US/docs/Web/CSS/align-self#Specifications
https://drafts.csswg.org/css-align-3/#propdef-align-self

Reply:
I don't know why we do this. "no" seems much clearer, and is what the specs actually say. But there might be a reason I don't understand that we do this.

Reply:
Oh, thank you! I've wondered what this means. You didn't have the full answer but at least you didn't know either. 😁